discount-details-block/src/BlockExtension.liquid

```diff
@@ -589,3 +244,5 @@ function parsePercentageMetafield(value) {
   }
 }
 
+{% endif %}
+
```

Comment:
the new line count of this file as a template is 💯 👌 

I know it still may be pending, but I approve of this 😁